### DISCUSSION TITLE
Wish List Item: Intro/Footer Text in Fabrik Options of Menu Item

### DISCUSSION THREAD

**Discussion by thommangooo** [trobfab/fabrik#541] [Ideas]
I'm not sure that this is the place for wish-list items, and i know it's not the time for making enhancements, but I'm gonna just park this here for now while I'm thinking of it.  Hope that's okay...

I really wish that, when creating a list menu item, within the Fabrik List Options dialog, there was a place for adding intro/footer text to be displayed above/below the list, just as there is within the list dialog.  Often, I find that I have a list and can re-use the same list for multiple contexts by adjusting the Fabrik Table Options - without needing to make copies of the list.  However, when doing this, I often want to provide a blurb of text tailored to the menu's context.    So, I'd like intro/footer text fields within the Fabrik List Options panel that would make text appear above/below a list when the list is displayed via the menu item.

**Comment by wezetel** [comment]
Hi thommangooo, why not just create menu items to display different articles and use the content plugin to include the list?

**Comment by wezetel** [comment]
Just retought: adjusting the table options might be not possible with the approach I mentioned. Needs a deeper look in the content plugin options

**Reply by thommangooo** [reply]
Hmmm.  I never even noticed this plug-in before.  This is cool!  I've made some components from scratch before which I might have made easier via this plug-in.  Thanks for mentioning it!

**Reply by wezetel** [reply]
This plugin is really useful. I use it in various situations. 
E.g. to add to a location list (with google maps elements) a Google maps vizualisation in the intro section, representing the (filtered) rows of the list. 
Another example: I have a generic fabrik contact form. And in an employee directory I have a calc field, which calls the contact form with the appropriate email address. According to my knowledge there are restrictions, so it is not possible to have two lists on one page (as far as I know)
